python/pubmed_crawler.py

```diff
@@ -200,19 +221,18 @@ def make_urls(url, accessions):
     return ", ".join(url_list)
 
 
-def scrap_info(pmids, curr_grants, consortium, name):
+def scrap_info(pmids, curr_grants, grant_view):
     """Create dataframe of publications and their pulled data.
 
     Returns:
         df: publications data
     """
 
-    first_colname = "CSBC PSON Center" if re.search(
-        r"csbc", name, re.I) else "Consortium Center"
-    columns = [first_colname, "Consortium", "PubMed", "Journal",
-               "Publication Year", "Title", "Authors", "Grant",
-               "SRX", "SRX Link", "SRP", "SRP Link", "dbGaP", "dbGaP Link",
-               "Data Location", "Synapse Location", "Keywords"]
+    columns = ["doi", "journal", "pubMedId", "pubMedUrl",
+               "publicationTitle", "publicationYear", "keywords",
+               "authors", "consortium", "grantId", "grantNumber",
+               "gseIds", "gseUrls", "srxIds", "srxUrls",
```

Comment:
Can you change `Ids` to `Accns`? NCBI is weird about what actually constitutes an "ID" (it's usually some sort of integer that you never see unless interacting with the API. The strings we use commonly — GSE1234, SRX5678, etc. are "accessions."

Reply:
Ah, that's good to know!